docs/howto/custom-template-tags.txt

```diff
@@ -498,6 +498,127 @@ you see fit:
     {% current_time "%Y-%m-%d %I:%M %p" as the_time %}
     <p>The time is {{ the_time }}.</p>
 
+.. _howto-custom-template-tags-simple-block-tags:
+
+Simple block tags
+-----------------
+
+.. method:: django.template.Library.simple_block_tag()
+
+Passing arguments may not be enough for some usages -- you may need to pass chunks of
+rendered template into your custom tags. This can be achieved using the
+``simple_block_tag`` helper function of ``django.template.Library``.
+``simple_block_tag`` functions similar to :meth:`~django.template.Library.simple_tag()`,
+however additionally passes a ``content`` argument which contains its rendered template
+content.
```

Comment:
```suggestion
If you need to pass a section of rendered template into your custom tag, this
can be achieved using the ``simple_block_tag`` helper function of
``django.template.Library``. ``simple_block_tag`` functions similar to
:meth:`~django.template.Library.simple_tag()`, but also passes a ``content``
argument which contains the rendered template content block contained by the
tag.
```
So I guess with this, the content itself is an argument and other arguments are not required (if that makes sense)